cmd/node/config/enableEpochs.toml

```diff
@@ -64,6 +64,9 @@
     # GovernanceEnableEpoch represents the epoch when governance is enabled
     GovernanceEnableEpoch = 5
 
+    # GovernanceFixesEnableEpoch represents the epoch when governance fixes are enabled
+    GovernanceFixesEnableEpoch = 6
```

Comment:
missing test in config/tomlConfig_test.go

Reply:
addewd